tests/flow/Makefile

```diff
@@ -5,11 +5,15 @@ test:
 ifeq ($(OS), Linux)
 	# regular tests on linux
 	@python3 -m RLTest --module ../../src/redisgraph.so $(TEST_ARGS)
+	# rerun persistency tests using AOF
+	@python3 -m RLTest --module ../../src/redisgraph.so --use-aof -t test_persistency $(TEST_ARGS)
 else
 # mac
 ifeq ($(VALGRIND),)
 	# no valgrind
 	@python3 -m RLTest --module ../../src/redisgraph.so $(TEST_ARGS)
+	# rerun persistency tests using AOF
+	@python3 -m RLTest --module ../../src/redisgraph.so --use-aof -t test_persistency $(TEST_ARGS)
```

Comment:
@MeirShpilraien 
Can `--use-aof` be specified by the actual test file? as an argument to `RLTest.Env` ?

Reply:
`Env` accepts a `useAof` argument, but two environments can't be live simultaneously, so we would need to duplicate the tests to run under both RDB and AOF.

Reply:
@jeffreylovitz @MeirShpilraien
how about we introduce 2 new classes both inheriting from `testGraphPersistency`?

```python
class testGraphPersistency(FlowTestsBase):
    def __init__(self, useAof):
        self.env = Env(decodeResponses=True, useAof=useAof)
        ...

class testGraphPersistencyAOF(testGraphPersistency):
    super(testGraphPersistency, True)
  
class testGraphPersistencyRDB(testGraphPersistency):
    super(testGraphPersistency, False)
```

Reply:
We can as well define a shared function that both tests calls to after creating the env, but I remember @jeffreylovitz wanted to avoid this. This is why I suggested this way..